manifests/examples/linux-bridge.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkState
+metadata:
+  name: node01
+spec:
+  managed: true
+  nodeName: node01
+  desiredState:
+    interfaces:
+      - name: eth1
+        type: ethernet
+        state: up
+      - name: br1
+        type: linux-bridge
+        state: up
+        bridge:
+          options:
+            group-forward-mask: 0
+            mac-ageing-time: 300
+            multicast-snooping: true
+            stp:
```

Comment:
I think we should drop stp and all other options other than basic. Let's not demo everything we support, but rather show that it is really easy to deploy a simple bridge

Reply:
Well, looks like nmstate is always returning even the default value set by the system, so we should pass even the expected defaults :-/, I suppose with policy this will change since we just pass the minimal.

Reply:
If that is the case, we have a bug in nmstate. nmstate should fill in the defaults. i.e. if we change their bridge test so there is no stp option, it should ignore it in the diff check and pass

Reply:
After talking at freenode #nmstate looks like they fill in the desired state with the default stuff from NetworkManager and it does stp enabled: true by default, I am changing that so we can disable it.

Also found a bug at our part 'omitempty' and false value was removing stp-hairpin-mode attribute making nmstatectl to fail, will push the fix.